sidebars.js

```diff
@@ -111,7 +111,7 @@ const sidebars = {
                 'build/tutorials/platform/transfer-avax-between-p-chain-and-c-chain',
                 'build/tutorials/platform/adjusting-gas-price-during-high-network-activity',
                 'build/tutorials/platform/sending-transactions-with-dynamic-fees-using-javascript',
-                'build/tutorials/platform/deploying-cross-chain-evm-bridge',
+                'build/tutorials/platform/subnets/deploying-cross-chain-evm-bridge',
```

Comment:
This should be moved to be under the subnets section. On the other hand, I was also thinking about the cross-reference. So far we don't have any doc showing up in two different sections yet.

Reply:
This is ok that this doc "deploying-cross-chain-evm-bridge" show up in two sections:  platform and subnet